columnq/tests/table_trino_test.rs

```diff
@@ -0,0 +1,68 @@
+#[cfg(feature = "database-trino")]
+mod trino_tests {
+    use columnq::table::TableSource;
+    use columnq::table::database::DatabaseLoader;
+    // Potentially add basic imports from connectorx if needed for mock/stub,
+    // but avoid pulling in too much if it complicates things without a live server.
+
+    #[tokio::test]
+    async fn test_trino_loader_construction() {
```

Comment:
this test doesn't seem to do anything? I am okay with merging without a test if it's too much work to setup trio in the CI.